<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMessageBox {
    background-color: black;
    border: 3px solid #00ff00;
	 font: 75 11pt &quot;White Rabbit&quot;;
}
QMessageBox QLabel {
    color: #00ff00;
}
QMessageBox QPushButton {
  background-color: black;
  border: 3px solid #00dd00;
  font: 75 16pt &quot;White Rabbit&quot;;
  color: #00dd00;
}
QMessageBox QPushButton:hover {
  border: 4px solid #00ff00;
  font: 75 17pt &quot;White Rabbit&quot;;
  color: #00ff00;
}

QDialog {
    background-color: black;
    border: 3px solid #00ff00;
	 font: 75 11pt &quot;White Rabbit&quot;;
}
QDialog QLabel {
    color: #00ff00;
}
QDialog QPushButton {
  background-color: black;
  border: 3px solid #00dd00;
  font: 75 16pt &quot;White Rabbit&quot;;
  color: #00dd00;
}
QDialog QPushButton:hover {
  border: 4px solid #00ff00;
  font: 75 17pt &quot;White Rabbit&quot;;
  color: #00ff00;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: black;
  height: 40px;
  border: 3px solid #00dd00;
  font: 75 16pt &quot;White Rabbit&quot;;
  color: #00dd00;
}
QPushButton:hover {
  border: 4px solid #00ff00;
  font: 75 17pt &quot;White Rabbit&quot;;
  color: #00ff00;
} 
QScrollBar:vertical {
     border: 2px solid #00ff00;
     background: black;
     width: 15px;
     margin: 22px 0 22px 0;
 }
 QScrollBar::handle:vertical {
	 border: 2px solid black;
     background: #00ff00;
 }
 QScrollBar::add-line:vertical {
     border: 2px solid #00ff00;
     background: black;
     height: 20px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
     border: 2px solid #00ff00;
     background: black;
     height: 20px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px solid black;
     width: 3px;
     height: 3px;
     background: #00ff00;
 }
 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="buttonWidget" native="true">
      <property name="styleSheet">
       <string notr="true">
QPushButton {
  background-color: black;
  height: 40px;
  border: 3px solid #00dd00;
  font: 75 16pt &quot;White Rabbit&quot;;
  color: #00dd00;
}
QPushButton:hover {
  border: 4px solid #00ff00;
  font: 75 17pt &quot;White Rabbit&quot;;
  color: #00ff00;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="images_train">
         <property name="text">
          <string>Images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="singleImage_train">
         <property name="text">
          <string>Single Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fonts_train">
         <property name="text">
          <string>Fonts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multi_lang">
         <property name="text">
          <string>KOR-ENG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transmit_button">
         <property name="text">
          <string>Transmit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="version_button">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="box_edit">
         <property name="styleSheet">
          <string notr="true">QPushButton.disabled {
  opacity: 0.6;
  cursor: not-allowed;
}</string>
         </property>
         <property name="text">
          <string>Box edit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="styleSheet">
         <string notr="true">QTextEdit {
border: 4px solid #00ff00;
font: 75 11pt &quot;White Rabbit&quot;;
background-color: black;
color: #00ff00;
}
</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="sysinfoLayout"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="quit_button">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
